PhotoMap/Features/Timeline/Controllers/TimelineViewController.swift

```diff
@@ -10,4 +10,58 @@ import Combine
 
 class TimelineViewController: BaseViewController {
     
+    // MARK: - Variables
+    private struct Constants {
+        static let cellIdentifier = "markerCell"
+    }
```

Comment:
You could improve that. Add extension:
```
extension UITableViewCell {
    static var identifier: String {
        return String(describing: self)
    }
}
```

And use this variable in dequeueReusableCell:
`MarkerCell.identifier`